### DISCUSSION TITLE
Pearl MMX snare hall effect

### DISCUSSION THREAD

**Discussion by 3hhh** [corrados/edrumulus#135] [Q&A]
I noticed that the [Pearl MMX samples for the snare](https://github.com/corrados/edrumulus/releases/tag/r0_8) produce an high pitched hall effect, which I find annoying as drumgizmo just plays multiple samples of it at the same time and it therefore amplifies into constant high-pitch humming. I guess the Pearl MMX snare simply has a large body to produce such an effect in reality, but probably it doesn't amplify that much in reality.

Is there a way to get rid of that effect or do I have to use different snare samples?

**Comment by corrados** [comment]
I think this is a Drumgizmo topic. I use ecasound to mix and equalize the individual tracks. You could apply some compression or gate or remove some low end. But I am no audio engineer.

**Comment by 3hhh** [comment]
Using `snare` and `snare_rimshot` samples with [`fade_out_percent = 100`](https://github.com/corrados/edrumulus/blob/43fee6396ca262b6b6c780b2eea9ba459b639b50/tools/create_drumgizmo_kit.py#L73) seems to make the snare samples shorter/more snappy and thus also helps with the hall effect.

**Reply by corrados** [reply]
That's great that you found such a simple solution.
